### PR TITLE
[WIP] Update Editor lib version (and fixes for file creation).

### PR DESCRIPTION
**Pull request by kegor**
Replicated changes from https://github.com/groupdocs-editor/GroupDocs.Editor-for-.NET-MVC/pull/15.
Required UI changes in: https://github.com/groupdocs-total/GroupDocs.Total-Angular/pull/194.

### CODE REVIEW

**Comment by bobkovalex** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           

Clones removed
==============
+ src/Products/Editor/Controllers/EditorApiController.cs  -4
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/groupdocs-total/GroupDocs.Total-for-NET-MVC/pullRequest?prid=5744964&bid=18795315)

**Comment by egorovpavel** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           

Clones removed
==============
+ src/Products/Editor/Controllers/EditorApiController.cs  -4
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/GroupDocs/GroupDocs.Total-for-NET-MVC/pullRequest?prid=5744965&bid=18795316)

File: src/Products/Editor/Controllers/EditorApiController.cs [hunk 1]
```diff
@@ -256,34 +262,87 @@ public HttpResponseMessage SaveFile(EditDocumentRequest postedData)
...
+                using (EditableDocument newFile = EditableDocument.FromMarkup(htmlContent, null))
+                {
+                    using (GroupDocs.Editor.Editor editor = new GroupDocs.Editor.Editor(saveFilePath))
+                    {
+                        dynamic saveOptions = this.GetSaveOptions(saveFilePath);
                         if (saveOptions is WordProcessingSaveOptions)
                         {
-                            saveOptions.EnablePagination = true;
+                            // TODO: leads to exception
+                            //saveOptions.EnablePagination = true;
```
Comment (egorovpavel) [inline src/Products/Editor/Controllers/EditorApiController.cs:340]: Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/manual/GroupDocs/GroupDocs.Total-for-NET-MVC/pullRequest?prid=5744965)

**Comment by egorovpavel** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 1
           

Clones removed
==============
+ src/Products/Editor/Controllers/EditorApiController.cs  -4
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/GroupDocs/GroupDocs.Total-for-NET-MVC/pullRequest?prid=5744965&bid=18795316)

**Comment by bobkovalex** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 1
           

Clones removed
==============
+ src/Products/Editor/Controllers/EditorApiController.cs  -4
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/groupdocs-total/GroupDocs.Total-for-NET-MVC/pullRequest?prid=5744964&bid=18795315)

File: src/Products/Editor/Controllers/EditorApiController.cs (hunk 1 above)
Comment (bobkovalex) [inline src/Products/Editor/Controllers/EditorApiController.cs:340]: Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/groupdocs-total/GroupDocs.Total-for-NET-MVC/pullRequest?prid=5744964)

**Comment by egorovpavel** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           

Clones removed
==============
+ src/Products/Editor/Controllers/EditorApiController.cs  -3
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/GroupDocs/GroupDocs.Total-for-NET-MVC/pullRequest?prid=5744965&bid=18795316)

**Comment by bobkovalex** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           

Clones removed
==============
+ src/Products/Editor/Controllers/EditorApiController.cs  -3
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/groupdocs-total/GroupDocs.Total-for-NET-MVC/pullRequest?prid=5744964&bid=18795315)

**Pull request merged**